src/subsystems.jl

```diff
@@ -240,8 +240,8 @@ function _check_topological_consistency(sys::System, component::Component)
                 sys,
                 component,
                 val,
-                "A component must be assigned to the same subsystems as its topological component(s).";
-                symmetric_diff = true,
+                "A component must be assigned to at least one subsystem as its topological component(s).";
```

Comment:
This check was not correct for a valid test system, but the code was not being exercised.

Reply:
I wonder why this test system was working